collections/Laud_Misc/MS_Laud_Misc_268.xml

```diff
@@ -52,63 +80,260 @@
+                        </supportDesc>
+                        <layoutDesc>
+                           <layout columns="2" writtenLines="51 52">Ruled in faint ink for 2 cols., 51-2 lines; <dimensions unit="mm" type="ruled">
+                              <height>205</height>
+                              <width>130</width>
+                           </dimensions> (inner bounding lines)</layout>
+                        </layoutDesc>
+                     </objectDesc>
+                     <handDesc hands="1">
+                        <handNote script="textualisNorthern">Northern textualis libraria; one hand; Italian characteristics, notably the abbreviation for 'qui'.</handNote></handDesc>
+                     <decoDesc>
+                        <decoNote type="flourInit">5-line puzzle initial in blue and red, fol. 43r; elsewhere 3-line (at the beginning of books) or 2-line initials in red flourished in light blue or dark blue flourished in red.</decoNote></decoDesc>
+                     <additions>
+                        <p>Running heads added; corrections to the text, extensive in places, by various hands, some German; nota marks in various hands, some also found in part III.</p>
+                     </additions>
+                  </physDesc>
+                  <history>
+                     <origin>
+                        <origDate calendar="Gregorian" notAfter="1310" notBefore="1290">13th century, late, or 14th century, early</origDate>
+                        <origPlace><country key="place_1000080">Italy (?)</country></origPlace>
+                     </origin>
+                  </history>
+               </msPart>
+               <msPart>
+                  <msIdentifier>
+                     <altIdentifier>
+                        <idno>MS. Laud Misc. 268 - part C (fols. 79-120)</idno>
+                     </altIdentifier>
+                  </msIdentifier>
+                  <msContents>
+                     <textLang mainLang="la">Latin</textLang>
+                     <msItem n="3">
+                        <locus>(fols. 79ra-85rb)</locus>
+                        <author key="person_44435485">Johannes Damascenus</author>
+                        <title key="work_7076">Elementarium dogmatum siue Institutio elementaris</title>
+                        <note>(tr. <persName role="trl" key="person_86920837">Robert Grosseteste</persName>)</note>
+                        <msItem><rubric>Incipit introductio dogmatum elementaris a voce sancti Johannis Damasceni ad
+                           Johannem sanctissimum episcopum <sic>Loadicie</sic></rubric>
+                        <incipit>In nomine Patris et Filii et Spiritus Sancti, unius trisypostate et adorate dietatis </incipit>
+                           <explicit>semper cum ipso esse et frui ipsius gaudia et regno nunc et <gap reason="editorial"/></explicit>
+                        <listBibl><bibl>CPG 8040</bibl>
```

Comment:
CPG numbers can be linked using <https://clavis.brepols.net/clacla/OA/link.aspx?clavis=CPG&number=8040>.